Owncloud iOs Client/Utils/UploadUtils.m

```diff
@@ -64,6 +65,9 @@ +(void) updateOverwritenFile:(FileDto *)file FromPath:(NSString *)path{
     DeleteFile *mDeleteFile = [[DeleteFile alloc] init];
     [mDeleteFile deleteItemFromDeviceByFileDto:file];
     
+    ManageThumbnails *manageThumbnails = [ManageThumbnails sharedManager];
```

Comment:
We should use directly the singleton not getting the singleton in a local instance. Just to use the same style on all the app (like the other singletons) :wink: